user/lxc/template.py

```diff
@@ -0,0 +1,40 @@
+pkgname = "lxc"
+pkgver = "6.0.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dapparmor=false",
+    "-Dman=false",
+    "-Dinit-script=sysvinit",
```

Comment:
this installs a etc/init.d/ script but that is not used since no openrc

Reply:
Not sure what to do about that, the two options are systemd and and sysinit.

Reply:
you can `self.uninstall` the etc/init.d folder

Reply:
```
Configuring lxc-apparmor-load using configuration
Run-time dependency systemd found: NO (tried pkgconfig)

config/init/systemd/meson.build:13:18: ERROR: Dependency "systemd" not found, tried pkgconfig

From meson configure, looks like we need something. 
```
  init-script          [systemd]              [systemd, sysvinit]    init script                               
```
```

Reply:
yea, i mean you can leave it as is but just delete the file after